gradle/libs.versions.toml

```diff
@@ -0,0 +1,231 @@
+androidx-lifecycle-runtime = { group = "androidx.lifecycle", name = "lifecycle-runtime", version.ref = "androidxLifecycle" }
+androidx-lifecycle-runtime-ktx = { group = "androidx.lifecycle", name = "lifecycle-runtime-ktx", version.ref = "androidxLifecycle" }
+androidx-lifecycle-viewmodel = { group = "androidx.lifecycle", name = "lifecycle-viewmodel", version.ref = "androidxLifecycle" }
+androidx-lifecycle-viewmodel-compose = { group = "androidx.lifecycle", name = "lifecycle-viewmodel-compose", version.ref = "androidxLifecycle" }
+androidx-lifecycle-viewmodel-ktx = { group = "androidx.lifecycle", name = "lifecycle-viewmodel-ktx", version.ref = "androidxLifecycle" }
+androidx-lifecycle-viewmodel-savedstate = { group = "androidx.lifecycle", name = "lifecycle-viewmodel-savedstate", version.ref = "androidxLifecycle" }
+androidx-navigation = { group = "androidx.navigation", name = "navigation-compose", version.ref = "androidxComposeNavigation" }
+androidx-percentLayout = { group = "androidx.percentlayout", name = "percentlayout", version.ref = "androidxPercentLayout" }
+androidx-preference = { group = "androidx.preference", name = "preference", version.ref = "androidxPreference" }
+androidx-recyclerView = { group = "androidx.recyclerview", name = "recyclerview", version.ref = "androidxRecyclerview" }
+androidx-swiperefreshlayout = { group = "androidx.swiperefreshlayout", name = "swiperefreshlayout", version.ref = "androidxSwipeRefreshLayout" }
+androidx-test-espresso-accessibility = { group = "androidx.test.espresso", name = "espresso-accessibility", version.ref ="androidxTestEspresso" }
+androidx-test-espresso-contrib = { group = "androidx.test.espresso", name = "espresso-contrib", version.ref ="androidxTestEspresso" }
+androidx-test-espresso-core = { group = "androidx.test.espresso", name = "espresso-core", version.ref ="androidxTestEspresso" }
+androidx-test-ext-junit = { group = "androidx.test.ext", name = "junit", version.ref ="androidxTestExtJunit" }
+androidx-test-rules = { group = "androidx.test", name = "rules", version.ref ="androidxTest" }
+androidx-test-runner = { group = "androidx.test", name = "runner", version.ref ="androidxTest" }
+androidx-test-ui-automator = { group = "androidx.test.uiautomator", name = "uiautomator", version.ref ="androidxTestUiAutomator" }
+androidx-viewPager2 = { group = "androidx.viewpager2", name = "viewpager2", version.ref ="androidxViewpager2" }
+androidx-webkit = {group = "androidx.webkit", name = "webkit", version.ref = "androidxWebkit"}
+androidx-workManager-runtime = { group = "androidx.work", name = "work-runtime", version.ref = "androidxWorkManager" }
+androidx-workManager-runtime-ktx = { group = "androidx.work", name = "work-runtime-ktx", version.ref = "androidxWorkManager" }
+androidx-workManager-workTesting = { group = "androidx.work", name = "work-testing", version.ref = "androidxWorkManager" }
```

Comment:
Minor (🔍): Consider updating these 3 `work` related dependencies above to a more straightforward naming, one that actually also matched the dependency signature:

```toml
androidx-work-runtime
androidx-work-runtime-ktx
androidx-work-testing
```

Reply:
fixed.